src/Reminders/Components/Reminder.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import type { Component } from "obsidian";
+  import Icon from "./Icon.svelte";
+  import Markdown from "./Markdown.svelte";
+  import type { Task } from "../../Task";
+
+  export let task: Task;
+  export let component: Component;
+  export let onRemindMeLater: (time: any) => void;
+  export let onDone: () => void;
+  export let onOpenFile: () => void;
+  // Do not set initial value so that svelte can render the placeholder `Remind Me Later`.
+  let selectedIndex: number;
+
+  export let laters: Array<number> = [];//Array<Later> = [];
+
+  function remindMeLater() {
+    const selected = laters[selectedIndex];
+    if (selected == null) {
+      return;
+    }
+    onRemindMeLater(selected); // selected.later
+  }
+</script>
+
+<main>
+  <h1 class="task-reminder-description">
+    <Markdown
+      markdown={task.description}
+      sourcePath={task.filename ?? "filename was null"} 
+      {component}
+    />
+  </h1>
+  <span class="task-file" on:click={onOpenFile}>
+    <Icon icon="link" />
+    {task.filename ?? "filename was null"}
+  </span>
+  <div class="task-actions">
+    <button class="mod-cta" on:click={onDone}>
+      <Icon icon="check-small" /> Mark as Done
+    </button>
+    <select
+      class="dropdown later-select"
+      bind:value={selectedIndex}
+      on:change={remindMeLater}
+    >
+      <!-- placeholder -->
+      <option selected disabled hidden>Remind Me Later</option>
+      <!-- options -->
+      {#each laters as i}
+        <option value={i} selected={selectedIndex === i}>{"later.label"}</option>
```

Comment:
There is something wrong with this display text. They all say `later.label`.

I suggest just not displaying "Remind me later" until it is working. And it's fine if that feature is implemented in a separate PR.

---

<img width="566" alt="image" src="https://user-images.githubusercontent.com/4840096/236662603-3425da53-dffb-4a16-bdb3-67297cffe4f1.png">